### ISSUE TITLE
To be addded

### ISSUE THREAD

**Issue by architsharm** [architsharm/XGboost-R#1]
1. Ask for factor columns
2. Ask for numeric column (check box/code)
3. Filter out anomalies
4. Ask for correlation allowed in individual columns
5. Ask for test train Seperately
6. Ask for label Column
7. Let user enter custom loss function and error function
8. Crossvalidation
